frontend-react/src/TelemetryService.ts

```diff
@@ -54,12 +57,77 @@ const createTelemetryService = () => {
 
 export const ai = createTelemetryService();
 
+export function getAppInsights() {
+    return appInsights;
+}
+
 export function getAppInsightsHeaders(): { [key: string]: string } {
     return {
-        "x-ms-session-id": getAppInsightsSessionId(),
+        "x-ms-session-id": appInsights?.context.getSessionId() || "",
     };
 }
 
-function getAppInsightsSessionId(): string {
```

Comment:
This doesn't really need to be a separate function since 1) it's only used in one place and 2) it wasn't even being exported.  🤷